### DISCUSSION TITLE
Storefront is throwing 404 Not found - nginx

### DISCUSSION THREAD

**Discussion by pOoOf** [saleor/saleor#7098] [Deployment]
I have deployed storefront here -> https://nf-saleor-storefront.herokuapp.com
and its backend here -> https://nf-saleor.herokuapp.com/graphql/
and dashboard here -> https://nf-saleor-dashboard.herokuapp.com

I just clicked deploy on Heroku button and followed the instructions, not sure what I am doing wrong here.
Can anybody tell me what could be the problem here or what I am doing wrong?
I have already tried deploying multiple times but couldn't get it to work.

Thanks

**Comment by gleniat** [comment]
Same here.
Just "nginx 404" on Heroku.
Working fine on localhost.
No errors in access log.

**Comment by sstrgh** [comment]
Same here

**Comment by krzysztofwolski** [comment]
Moving deployment issues to community section.

**Comment by pOoOf** [comment]
can anyone help here?
